pkg/dns/aws/dns.go

```diff
@@ -309,6 +310,15 @@ func (m *Manager) updateAlias(domain, zoneID, target, targetHostedZoneID, action
 		},
 	})
 	if err != nil {
+		if action == "DELETE" {
```

Comment:
I'd leave the error interpretation to the caller instead of inlining it here... also, there's a `deleteAction` constant

Reply:
Thanks for the feedback. I'll move to the caller.